test/Infrastructure.IntegrationTest/AdminConsole/Repositories/OrganizationEnableCollectionEnhancementTests.cs

```diff
@@ -11,7 +11,7 @@ namespace Bit.Infrastructure.IntegrationTest.AdminConsole.Repositories;
 
 public class OrganizationEnableCollectionEnhancementTests
 {
-    [DatabaseTheory, MssqlDatabaseData]
```

Comment:
Please delete the `MssqlDatabaseData` class, it was really just a hack because we didn't have an EF implementation.